checkov/sast/checks/base_registry.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import logging
+import os
+import yaml
+from typing import List, Any, Optional
+from checkov.common.bridgecrew.check_type import CheckType
+from checkov.common.checks.base_check_registry import BaseCheckRegistry
+from checkov.sast.enums import SastLanguages
+
+CHECKS_POSSIBLE_ENDING = [".yaml", ".yml"]
```

Comment:
import `CHECKS_POSSIBLE_ENDING` from `commons.checks_infra.registry`

Reply:
Why? this is not related to graph checks

Reply:
They share the check file format, and unless we change the SAST check format it's a clear code duplication..